airbyte-integrations/connectors/source-stripe/source_stripe/schemas/subscription_schedule.json

```diff
@@ -0,0 +1,240 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Generated schema for Root",
+    "type": "object",
+    "properties": {
```

Comment:
The schema is not correct, @btkcodedev please check others connector and correct it.